### ISSUE TITLE
This repo is in DESPERATE need of documentation.

### ISSUE THREAD

**Issue by 10by10pixel** [IcyDefiance/Veuwer#19]
I think this repo would get way more help and pull requests if it was even slightly documented. At the *very least* I would add some build instructions to the readme so people can make test installs and help fix bugs accordingly.

**Comment by IcyDefiance** [comment]
Agreed. Problem is no one can run it locally right now without access to an AWS account, and there's been more urgent things to work on so I haven't had time to implement support for local storage.

Once I get time to do that, a bit of organization and documentation will come soon afterward, and I'll leave this issue open as a reminder. Things just need to calm down a bit first.
